frontends/web/src/routes/lightning/send/send.tsx

```diff
@@ -158,53 +159,59 @@ export function Send({ accounts, code }: Props) {
     switch (step) {
     case 'select-invoice':
       return (
-        <Grid col="1">
-          <Column>
-            <QrCodeInput
-              title={t('send.scanQR')}
-              label={t('lightning.send.rawInput.label')}
-              placeholder={t('lightning.send.rawInput.placeholder')}
-              inputError={rawInputError}
-              onInputChange={onRawInputChange}
-              value={rawInput}
-              activeScanQR={activeScanQr}
-              parseQRResult={setRawInput}
-              onChangeActiveScanQR={onChangeActiveScanQr}
-            />
-            <ColumnButtons className="m-top-default m-bottom-xlarge" inline>
-              <Button primary onClick={parseInput} disabled={busy}>
-                {t('button.send')}
-              </Button>
-              <Button secondary onClick={back}>
-                {t('button.back')}
-              </Button>
-            </ColumnButtons>
-          </Column>
-        </Grid>
+        <View fitContent>
+          <ViewContent>
+            <Grid col="1">
+              <Column>
+                {/* this flickers quickly, as there is 'SdkError: Generic: Breez SDK error: Unrecognized input type' when logging rawInputError */}
+                { rawInputError && (
+                  <Status type="warning">{rawInputError}</Status>
+                )}
+                <ScanQRVideo onResult={onInvoice} />
+                {/* Note: unfortunatelly we probably can't read from HTML5 clipboard api directly in Qt/Andoird WebView */}
```

Comment:
Seems so. Maybe an Input or Dialog to paste the clipboard into?

Reply:
Input field would work ofc, but  I think we want it as a button. Discussed with jokoono (not sure what his GH is), and left the button doing-nothing as a placeholder.